src/plugins/unified_histogram/public/services/lens_vis_service.ts

```diff
@@ -455,13 +455,7 @@ export class LensVisService {
     queryParams: QueryParams;
   }): Suggestion | undefined => {
     const { dataView, query, timeRange } = queryParams;
-    if (
-      dataView.isTimeBased() &&
-      query &&
-      isOfAggregateQueryType(query) &&
-      getAggregateQueryMode(query) === 'esql' &&
```

Comment:
ℹ️ Irrelevant with this PR, I just removed it as is not needed anymore (we removed SQL)